tests/shared/src/test/scala-2/zio/schema/MetaSchemaSpec.scala

```diff
@@ -123,28 +131,30 @@ object MetaSchemaSpec extends ZIOSpecDefault {
             )(_.asInstanceOf[SchemaGen.Arity2])(_.asInstanceOf[Any])(_.isInstanceOf[Any])
           )
         val expectedAst =
-          MetaSchema.Sum(
+          ExtensibleMetaSchema.Sum(
             TypeId.Structural,
             path = NodePath.root,
             cases = Chunk(
-              "type1" -> MetaSchema.Product(
+              "type1" -> ExtensibleMetaSchema.Product(
                 id = TypeId.parse("zio.schema.SchemaGen.Arity1"),
                 path = NodePath.root / "type1",
                 fields = Chunk(
-                  "value" -> MetaSchema.Value(StandardType.IntType, NodePath.root / "type1" / "value")
+                  "value" -> ExtensibleMetaSchema
+                    .Value[DynamicValue :: End](StandardType.IntType, NodePath.root / "type1" / "value")
                 )
               ),
-              "type2" -> MetaSchema.Product(
+              "type2" -> ExtensibleMetaSchema.Product(
                 id = TypeId.parse("zio.schema.SchemaGen.Arity2"),
                 path = NodePath.root / "type2",
                 fields = Chunk(
-                  "value1" -> MetaSchema.Value(StandardType.StringType, NodePath.root / "type2" / "value1"),
-                  "value2" -> MetaSchema.Product(
+                  "value1" -> ExtensibleMetaSchema
+                    .Value[DynamicValue :: End](StandardType.StringType, NodePath.root / "type2" / "value1"),
+                  "value2" -> ExtensibleMetaSchema.Product(
                     id = TypeId.parse("zio.schema.SchemaGen.Arity1"),
                     path = NodePath.root / "type2" / "value2",
                     fields = Chunk(
-                      "value" -> MetaSchema
-                        .Value(StandardType.IntType, NodePath.root / "type2" / "value2" / "value")
+                      "value" -> ExtensibleMetaSchema
+                        .Value[DynamicValue :: End](StandardType.IntType, NodePath.root / "type2" / "value2" / "value")
```

Comment:
It may be helpful to have method-based constructors that can fill in `DynamicValue :: End` since it's so common. e.g. `def value(...): MetaSchema[...]`.

Reply:
I mean, in the companion object of `ExtensibleMetaSchema`, of course.

Reply:
Almost did it :) but then I thought normal usage of `MetaSchema` does not really require dealing with its constructors, it is just this test suite. Normally you just use it as a serializable representation of a schema and not really manipulating it.